abs.yaml

```
@@ -6,3 +6,5 @@ extra_labels_for_os:
   # We aren't strictly tied to ventura but we do get kernel panics on
   # the builders if we're not.
   osx-arm64: [ventura]
+  osx-64: [ventura]
+  
```

```suggestion

build_env_vars:
  ANACONDA_ROCKET_ENABLE_PY313 : yes
```

To enable 3.13 build. And I don't think we have a ventura label for the osx-64 builder.

---

Also, can you remvove the build_parameters section? It's disabling the overlinking checks.

---

Multiple tests were timing out without `osx-64: [ventura]`.
It might have been a coincidence they passed after I've added this?